gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -178,7 +178,8 @@ private static void configureErrorProneOptions(
 
         errorProneOptions.setEnabled(true);
         errorProneOptions.setDisableWarningsInGeneratedCode(true);
-        errorProneOptions.setExcludedPaths(project.getBuildDir().getAbsolutePath() + "/.*");
+        errorProneOptions.setExcludedPaths(
+                String.format("%s/(build|src/generated.*)/.*", project.getProjectDir().getAbsolutePath()));
```

Comment:
Thoughts on including `.*/generated/.*`? It's possible applications have packages named `generated`, but that doesn't make a great deal of sense for non-generated code.

Reply:
I'd prefer to be more strict to begin with since being overly lenient could lead to false negatives.